versions/latest/modules/en/pages/integrations/integrations.adoc

```diff
@@ -6,7 +6,16 @@ Prime also offers options for production support, as well as add-ons to your sub
 
 To learn more and get started with Rancher Prime, please visit https://www.rancher.com/quick-start[this page].
 
-import DocCardList from '@theme/DocCardList';
-import { useCurrentSidebarCategory } from '@docusaurus/theme-common/internal';
+To learn more about Rancher's ecosystem, refer to the following integrations:
 
-<DocCardList items={useCurrentSidebarCategory().items.slice(0,9)} />
```

Comment:
`(0,9)` means there are 10 items, but there are 11 items in your list. Something extra has been added.